Orange/data/io.py

```diff
@@ -369,7 +369,8 @@ def _equal_length(lst):
 
             if type_flag in StringVariable.TYPE_HEADERS:
                 coltype = StringVariable
-
+                values = [None if isinstance(i, float)
```

Comment:
Why not `''` (an empty string) instead of `None`? Since `''` in data stands for a missing value, there's no way to discern missing strings from non-missing empty strings anyway.

Could also avoid another pass through the column if `type_flag` was set before `orig_values` a couple lines above.